cpp/include/cuml/tree/algo_helper.h

```diff
@@ -31,4 +32,12 @@ enum CRITERION {
   CRITERION_END,
 };
 
+/** check for treelite runtime API errors and assert accordingly */
```

Comment:
By putting this check macro in this header file, we will now have to also ship the treelite header files along with ours! Any reason for moving this macro from `decisiontree_impl.h` to here?

Reply:
don't remember anymore, it works without all this, so reverting. thanks!